packages/fiori/src/SideNavigationSelectableItemBase.ts

```diff
@@ -90,6 +92,7 @@ class SideNavigationSelectableItemBase extends SideNavigationItemBase {
 	 * - `_search`
```

Comment:
I would change "Notes" to "Values" or "Possible values" and add "Notes" below them.
Also, I would remove the bold from "This property must only be used when the `href` property is set."

I would change "Items that have a set href and target set to <code>_blank</code> should not have children." to "Items that have a defined "href" and a "target" attribute set to <code>_blank</code> should not be selectable" 
and make it not bold.

Should we add "Items that have a defined "href" and a "target" attribute set to <code>_blank</code> should not be selectable" also to the "href" property?

Reply:
I removed "This property must only be used when the href property is set."